tmt/export.py

```diff
@@ -91,18 +105,17 @@ def convert_manual_to_nitrate(test_md):
     as html strings.
     """
 
-    values = []
-    sections_headings = {}
-    for _ in list(tmt.base.SECTIONS_HEADINGS.values()):
-        values += _
-    for _ in values:
-        sections_headings[_] = []
+    sections_headings: SectionsHeadingsType = {
+        heading: []
+        for heading_list in list(tmt.base.SECTIONS_HEADINGS.values())
```

Comment:
I'd say `list()` is completely unnecessary, one can iterate `.values()` directly.